src/main/java/cat/udl/eps/entsoftarch/webingogeiapi/handler/CardEventHandler.java

```diff
@@ -7,17 +7,28 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.rest.core.annotation.HandleAfterCreate;
 import org.springframework.data.rest.core.annotation.HandleBeforeCreate;
+import org.springframework.data.rest.core.annotation.RepositoryEventHandler;
+import org.springframework.stereotype.Component;
+
+@Component
+@RepositoryEventHandler
 
 public class CardEventHandler {
     final Logger logger = LoggerFactory.getLogger(Card.class);
 
     @Autowired
     CardRepository CardRepository;
 
+    public CardEventHandler(CardRepository CardRepository) {
```

Comment:
Constructor not required, @Autowired already injects an instance of cardRepository. Moreover, the recommended practice is to use lower case for variables, thus `@Autowired CardRepository cardRepository;`